Dicee Challenge - Starting Files/README.md

```diff
@@ -0,0 +1,6 @@
+This is a begginer Friendly Mini project using Html,CSS and Javascript. In this project two players are present, when you click the start button game will start,Dices will move and according to their score which one will be greater it will win the game and it start's again. Here javascript Math.random function and DOM manipulation is Used ,It is a begginer project to make to skill better.![Screenshot (33)](https://user-images.githubusercontent.com/100128956/195383106-b073e8fe-7f99-4dbe-8512-f8e393dcb2c4.png)
+
```

Comment:
Please add more details and explanation of how you have used various technologies here